src/components/pages/tasks/TasksPage.tsx

```
@@ -0,0 +1,122 @@
+import { TaskGrid } from '@/components/tasks/grid/TaskGrid';
+import { MainLayout } from '@/app/(protected)/MainLayout';
+import { AnimatedTransition } from '@/components/ui/animation/AnimatedTransition';
+import { Button } from '@/components/ui/button/Button';
+import { usePagination } from '@/hooks/usePagination';
+import styles from './TasksPage.module.scss';
+
+const TasksPage = () => {
+  const { tasks, totalPages, loading, fetchTasks, error } = useTasks();
+  const toast = useToastContext();
+  const [searchQuery, setSearchQuery] = useState('');
+  const [isSearching, setIsSearching] = useState(false);
```

ESLint

[error] 'isSearching' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)